policies/elections.tex

```diff
@@ -345,15 +358,14 @@ \subsubsection{Petitions}
 If the petition is determined to be invalid, the Elections Committee shall return it to the submitter with an explanation of why it was deemed invalid.
 
 A petition for a referendum on a nonsensical or useless resolution shall not be accepted.
-If a petition contains multiple resolutions, some of which are nonsensical or useless, the petition shall still be accepted for the remaining
-resolution or resolutions.
+If a petition contains multiple resolutions, some of which are nonsensical or useless, the petition shall still be accepted for the remaining resolution or resolutions.
 
 \subsubsection{Schedule}
 When a referendum is to be held, the Elections Committee or Council shall set the schedule.
-It shall consist of a campaigning period of at least 5 days, and a voting period of 2 days.
+It shall consist of a campaigning period containing at least 5 business days, and a voting period of 2 business days.
```

Comment:
It shall consist of a campaigning period containing at least 5 business days, and a voting period containing 2 to 5 business days.